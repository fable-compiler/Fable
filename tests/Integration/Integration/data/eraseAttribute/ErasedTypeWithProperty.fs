module ErasedTypeWithProperty

open Fable.Core

[<Erase>]
type internal LanguageInjectionAttribute() =

    [<Erase>]
    member val Prefix = "" with get, set

// Force Fable to generate a file, otherwise because everything is erased, it will not generate anything
let a = 0
