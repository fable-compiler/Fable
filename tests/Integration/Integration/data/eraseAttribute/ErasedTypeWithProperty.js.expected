
export const a = 0;

